src/components/ChatWindow/ChatWindowHeader/ChatWindowHeader.tsx

```diff
@@ -21,6 +21,8 @@ const useStyles = makeStyles(() =>
     closeChatWindow: {
       cursor: 'pointer',
       display: 'flex',
+      background: 'transparent',
+      border: '0',
```

Comment:
Nit: I think adding `padding: '0.4em'` helps a little. It makes the blue border square when it is focused. Previously it was a little rectangular.

![image](https://user-images.githubusercontent.com/12685223/110825146-f8e9d280-8250-11eb-9780-b1ca52fa3918.png)